### ISSUE TITLE
[DOCS] Update all images to be transparent

### ISSUE THREAD

**Issue by vinomaster** [finos/a11y-theme-builder#49]
## Problem/Concern

We use Material for MkDocs which supports light and dark mode toggling.

Currently images used across docs (specifically in Overview (/motivation) use a white background and there do not work well in Dark Mode.

## Proposed Solution
Update all images to have transparent background.

**Comment by aaronreed708** [comment]
It isn't going to be as easy as changing the background color of the image, unfortunately.  For example, consider what this image would look like in dark mode:  https://discoverfinancial.github.io/a11y-theme-builder/_images/gaad-a11y-e2e.png

It has dark text in it, as well as dark inking for the people at the bottom and grays in the Design, Develop, Code, Interact section of the image.  All of these would be practically invisible with transparent background in dark mode.

Here is some guidance on how we should handle images in light and dark mode in mkdocs: https://squidfunk.github.io/mkdocs-material/reference/images/#light-and-dark-mode

Basically we need dark-mode versions of our images, especially those that don't have high contrast with dark colors.

These images probably need to be re-worked to create dark theme specific versions
- [ ] _images/statista-2023-digital-population-worldwide.png
- [ ] _images/gaad-a11y-e2e.png
- [ ] _images/unified-design.png
- [ ] _images/layers.png
- [ ] _images/architecture.png

Perhaps also the logo.png from Theme Builder itself.  It has a white background with dark lines.

**Comment by brycecurtis** [comment]
To enable an image for dark mode, a filter that darkens the image is required.  @lwnoble will provide the overlay settings/css.

**Comment by lwnoble** [comment]
for the css

img {
opacity: .68;
}

That should do it!

**Comment by aaronreed708** [comment]
I introduced a css file that will be included that will change the opacity (as described above) for all images contained in all elements with the "slate" (aka dark mode) scheme applied to it.
